templates/settings.py.erb

```diff
@@ -60,6 +60,7 @@ REST_FRAMEWORK__DEFAULT_AUTHENTICATION_CLASSES = (
 ALLOWED_IMPORT_PATHS = <%= scope['pulpcore::allowed_import_path'] %>
 ALLOWED_EXPORT_PATHS = <%= scope['pulpcore::allowed_export_path'] %>
 ALLOWED_CONTENT_CHECKSUMS = <%= scope['pulpcore::allowed_content_checksums'] %>
+ALLOW_AUTOMATIC_UNSAFE_ADVISORY_CONFLICT_RESOLUTION = <%= scope['pulpcore::allow_automatic_unsafe_advisory_conflict_resolution'] %>
```

Comment:
You should convert this to Python booleans. See a few lines below for `CACHE_ENABLED`, or use `to_python()`.